### PR TITLE
feat(backend): get page of payment pointers

### LINKED ISSUE #1344: Add GraphQL admin API query to fetch list of payment pointers
Similar to how we can query peers and assets, we can add a query to the admin API that will use pagination for providing a list of payment pointers.

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- adds possibility to get page of payment pointers via Admin API

## Context

- fixes #1344 

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass

### CODE REVIEW

File: packages/backend/src/shared/pagination.test.ts [hunk 1]
```diff
@@ -58,6 +59,52 @@ describe('Pagination', (): void => {
     )
   })
   describe('getPageInfo', (): void => {
+    describe('payment pointers', (): void => {
```
Comment (mkurapov) [inline packages/backend/src/shared/pagination.test.ts:62]: I'm going to have a similar comment that I left on @BlairCurrey 's PR:
- https://github.com/interledger/rafiki/pull/1454#discussion_r1204616058

I think we should have pagination tests for payment pointers live in the `PaymentPointerService` tests

Comment (sabineschaller) [reply packages/backend/src/shared/pagination.test.ts:62]: Happy to change that. I also found it weird that it was where it was but I put it there for consistency.

File: packages/backend/src/open_payments/payment_pointer/service.test.ts [hunk 2]
```diff
@@ -214,6 +215,49 @@ describe('Open Payments Payment Pointer Service', (): void => {
     })
   })
 
+  describe('Payment Pointer pagination', (): void => {
```
Comment (BlairCurrey) [inline packages/backend/src/open_payments/payment_pointer/service.test.ts:218]: Does this supersede the need for the other `getPageTests` call? In `assets` and `peers` we call  `getPageTests` (with different options) in both the resolver and the service.

I was expecting something like this here too:

```ts
    getPageTests({
      createModel: () => createPaymentPointer(deps),
      getPage: (pagination?: Pagination) =>
        paymentPointerService.getPage(pagination)
    })
```

Comment (BlairCurrey) [reply packages/backend/src/open_payments/payment_pointer/service.test.ts:218]: I really should have highlighted the line below. I'm referring the cases in the `test.each`.

Comment (sabineschaller) [reply packages/backend/src/open_payments/payment_pointer/service.test.ts:218]: Oh, I missed the ones for the service. Thank you!

These are actually 2 different sets of tests, the ones for the resolvers (packages/backend/src/graphql/resolvers/page.test.ts) and the ones for the services (packages/backend/src/shared/baseModel.test.ts).

I'll add the service tests.

**Review by BlairCurrey — APPROVED** [review]

**Pull request merged**